src/navigators/TabNavigator.tsx

```diff
@@ -78,30 +39,51 @@ export function TabNavigator() {
       inactiveColor={theme.colors.onSurfaceDisabled}
       labeled
     >
-      {navigationTabs
-        ? navigationTabs?.map((tab) => (
-            <Tab.Screen
-              name={tab.name}
-              component={tab.component}
-              key={tab.name}
-              options={{
-                tabBarLabel: tab.options.tabBarLabel,
-                tabBarIcon: tab.options.tabBarIcon,
-                headerShown: false,
-              }}
-            />
-          ))
-        : defaultTabs()}
+      <Tab.Screen
+        name={tabs?.[0].name ?? 'HomeTab'}
+        component={tabs?.[0].component ?? HomeStack}
+        key={tabs?.[0].name ?? 'HomeTab'}
+        options={{
+          tabBarLabel: tabs?.[0].label ?? t('tabs-home', 'Home'),
+          tabBarIcon: tabs?.[0].icon ?? Home,
+          tabBarColor: tabs?.[0].color ?? 'red',
+          headerShown: tabs?.[0].headerShown ?? false,
+        }}
+      />
+      <Tab.Screen
+        name={tabs?.[1].name ?? 'NotificationsTab'}
+        component={tabs?.[1].component ?? NotificationsStack}
+        key={tabs?.[1].name ?? 'NotificationsTab'}
+        options={{
+          tabBarLabel:
+            tabs?.[1].label ?? t('tabs-notifications', 'Notifications'),
+          tabBarIcon: tabs?.[1].icon ?? Bell,
+          tabBarColor: tabs?.[1].color,
+          headerShown: tabs?.[1].headerShown ?? false,
+        }}
+      />
+      <Tab.Screen
+        name={tabs?.[2].name ?? 'SettingsTab'}
+        component={tabs?.[2].component ?? SettingsStack}
+        key={tabs?.[2].name ?? 'SettingsTab'}
+        options={{
+          tabBarLabel: tabs?.[2].label ?? t('tabs-settings', 'Settings'),
+          tabBarIcon: tabs?.[2].icon ?? Settings,
+          tabBarColor: tabs?.[2].color,
+          headerShown: tabs?.[2].headerShown ?? false,
```

Comment:
This totally works, but it also feels somewhat fragile around how many items are in `tabs`, right?  Seems ok for a first pass, but we want to make this more dynamic/responsive to the `tabs` count passed in over time, right?

Reply:
Yeah, that's a great point, and I think I see a clean way to do it and also consolidate the "AdditionalTabs" into it.